Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -145,10 +147,19 @@ public NodejsProjectNode(NodejsProjectPackage package)
         }
 
         private void TryToAcquireTypings(IEnumerable<string> packages) {
+            bool isNewTypingsFolder = !Directory.Exists(Path.Combine(this.ProjectHome, "typings"));
             if (ShouldAcquireTypingsAutomatically && TypingsAcquirer != null) {
                 TypingsAcquirer
                     .AcquireTypings(packages, null /*redirector*/)
-                    .ContinueWith(x => x);
+                    .ContinueWith(x => {
+                        if (NodejsPackage.Instance.IntellisenseOptionsPage.ShowTypingsInfoBar &&
```

Comment:
For the future: if the acquisition fails, do we also want to show a message of some kind? Typing acquisition failures will result in an unexpectedly degraded intellisense experience.